ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/legaladvisor/CCDOrderDirection.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.cmc.ccd.domain.legaladvisor;
+
+import lombok.Builder;
+import lombok.Value;
+
+import java.time.LocalDate;
+
+@Value
+@Builder
+public class CCDOrderDirection {
+    private CCDOrderDirectionType extraOrderDirection;
+    private String otherDirection;
+    private LocalDate sendBy;
```

Comment:
Is this order creation date or the date it should be posted by?

Reply:
It should be the date it should be posted by